src/runtime/help-pmix-runtime.txt

```diff
@@ -40,3 +40,20 @@ a fatal situation, but may negatively impact your launch performance.
 If you feel you have received this warning in error, or wish to ignore
 it in the future, you can disable it by setting the PMIx MCA parameter
 "pmix_suppress_missing_data_warning=1"
+#
+[no-plugins]
+We were unable to find any usable plugins for the %s framework. This PMIx
+framework requires at least one plugin in order to operate. This can be caused
+by any of the following:
+
+* we were unable to build any of the plugins due to some combination
+  of configure directives and available system support
+
+* no plugin was selected due to some combination of MCA parameter
+  directives versus built plugins (i.e., you excluded all the plugins
+  that were built and/or could execute)
+
+* the PMIX_INSTALL_PREFIX is pointing to a location that doesn't include
```

Comment:
I'm not sure I comprehend the concern. We have no way of determining who set the envar, or why they set it - all we can say is that the envar is set and pointing to an incorrect location. What more would you have us say?